pom.xml

```diff
@@ -293,6 +293,13 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Transformer generation project profile -->
+        <profile>
+            <id>transformer-generator</id>
+            <modules>
+                <module>bull-transformation-generator</module>
```

Comment:
Maybe we could rename the module to `bull-transformer-generator`, like the project and branch?

Reply:
I fully agree. The change as been done